janus_core/processing/correlator.py

```diff
@@ -278,7 +284,10 @@ def get(self) -> tuple[Iterable[float], Iterable[float]]:
         lags : Iterable[float]]
             The correlation lag times t'.
         """
-        return self._correlator.get()
+        if self._correlators:
+            _, lags = self._correlators[0].get()
+            return np.mean([cor.get()[0] for cor in self._correlators], axis=0), lags
```

Comment:
Does calling `get` an extra time for the lags not also recalculate non-trivial things that we don't need? Maybe we want a separate `get_lags` for this?

Reply:
Also that the lags are recomputed each get, but we only need them once. They are split now